docs/4.3.yaml

```diff
@@ -0,0 +1,79 @@
+site_name: Gravitational Teleport
+site_url: https://gravitational.com/teleport/docs
+repo_url: https://github.com/gravitational/teleport
+site_description: SSH and Kubernetes Gateway for Clusters and Teams
+site_author: Gravitational Inc
+copyright: Gravitational Inc, 2016-20
+
+# output directory:
+site_dir: ../build/docs/4.3
+docs_dir: "4.3"
+
+theme:
+    name: null
+    custom_dir: 'theme/'
+    include_search_page: true
+markdown_extensions:
+  - markdown_include.include:
+      base_path: '../'
+  - admonition
+  - def_list
+  - footnotes
+  - toc:
+      marker: '[TOC]'
+      toc_depth: 2
+extra_css: []
+plugins:
+    - search:
+        separator: '[\s\-\.]'
+    - markdownextradata: {
+    }
+extra_javascript: []
+extra:
+    version: 4.3
+    teleport:
+        version: 4.3
```

Comment:
Should be `4.3.0` if we're going to do this as it needs to be the number that'll be in the `get.gravitational.com` links

Reply:
good catch.